### DISCUSSION TITLE
Huskchat support?

### DISCUSSION THREAD

**Discussion by Kasviss** [PotatoCraft-Studio/QuickShop-Reremake#383] [Ideas]
Can you add Huskchat support to the Quickshop plugin? That is, all conversations go through a proxy server, not normally through Minecraft's normal conversation. The problem is that if I put a message, for example: when the extension asks what price you want for the product, I send 1 to the conversation, but it doesn't do anything. 
![image](https://github.com/PotatoCraft-Studio/QuickShop-Reremake/assets/139772321/02827c7e-ea8d-4bb5-bb18-c93c400f025b)

**Comment by Kasviss** [comment]
Link for Huskchat: https://www.spigotmc.org/resources/huskchat.94496/

**Comment by sandtechnology** [comment]
please contact its author, we just handle and cancel chat event currently, nothing wrong in our side

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Kasviss** [reply]
So what should I say to him?
